tests/installation/bootloader_svirt.pm

```diff
@@ -57,14 +57,14 @@ sub run {
     my $arch       = get_var('ARCH');
     my $vmm_family = get_required_var('VIRSH_VMM_FAMILY');
     my $vmm_type   = get_required_var('VIRSH_VMM_TYPE');
-
-    my $svirt = select_console('svirt');
-    my $name  = $svirt->name;
+    my $svirt      = select_console('svirt');
```

Comment:
not `console('svirt')` ?

Reply:
I have found it in the test code. Mainly the line was just touched by my tidy script. :)

Reply:
Ok. I was just wondering why you use it different as in `data_integrity_utils.pm`

Reply:
Honestly, I do not know. I guess you need to raise the question to Joaquin as it his code. I have just added an argument in case of vmware.